src/background/alarm-utils.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+export class AlarmUtils {
+    protected alarmCallbacks = {};
+    constructor(protected alarmAPI?) {
+        this.alarmAPI ||= chrome.alarms;
```

Comment:
nit: this could be simplified as
```suggestion
    constructor(protected readonly alarmAPI = chrome.alarms) {
```

Reply:
ooh good call I'm going to try this locally and make sure the chrome global doesn't upset tests but this looks much nicer.